<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BAPANDEVI1" ignored="0" name="TestNG Demo(failed)" tests="1" failures="1" timestamp="29 Nov 2018 14:17:22 GMT" time="2.193" errors="0">
  <testcase name="timeout" time="2.072" classname="TestNG.TimeOutTestNgDemo">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method TestNG.TimeOutTestNgDemo.timeout() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestNG.TimeOutTestNgDemo.timeout() didn't finish within the time-out 2000
at java.io.FileOutputStream.writeBytes(Native Method)
at java.io.FileOutputStream.write(Unknown Source)
at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
at java.io.BufferedOutputStream.flush(Unknown Source)
at java.io.PrintStream.write(Unknown Source)
at sun.nio.cs.StreamEncoder.writeBytes(Unknown Source)
at sun.nio.cs.StreamEncoder.implFlushBuffer(Unknown Source)
at sun.nio.cs.StreamEncoder.flushBuffer(Unknown Source)
at java.io.OutputStreamWriter.flushBuffer(Unknown Source)
at java.io.PrintStream.write(Unknown Source)
at java.io.PrintStream.print(Unknown Source)
at java.io.PrintStream.println(Unknown Source)
at TestNG.TimeOutTestNgDemo.timeout(TimeOutTestNgDemo.java:14)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- timeout -->
</testsuite> <!-- TestNG Demo(failed) -->
